package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@honey-finance/sdk",
-  "version": "1.1.2",
+  "version": "1.1.3",
```

Comment:
1.1.3 was already deployed 1 month ago, should this be changed? @bbroeking 
https://www.npmjs.com/package/@honey-finance/sdk?activeTab=versions